src/main/java/frc/team4909/robot/sensors/Stream.java

```diff
@@ -1,103 +1,76 @@
+      camera1.setResolution(160, 120);    /**********************  Set   ************** */
+      camera0.setFPS(21);                 /********************  Camera  ************** */  
+      camera1.setFPS(16);                 /*************  Specifications    *********** */
+      camera2.setResolution(160, 120);    /******************************************** */
+      camera3.setResolution(160, 120);    /******************************************** */
+      camera2.setFPS(10);                 /******************************************** */
+      camera3.setFPS(10);                 /******************************************** */
+      //////VideoSink intakeServer = CameraServer.getInstance().getServer("Intake Cam");
+      CameraServer intakeServer = CameraServer.getInstance();//.addCamera(intakeServer);
+      ///////SmartDashboard.putData(intakeServer);
+      //CvSink cvSink1 = CameraServer.getInstance().getVideo();//CvSource outputStream = CameraServer.getInstance().putVideo("Source 1", 160, 120);//CvSink cvSink2 = CameraServer.getInstance().getVideo();//CvSource outputStream2 = CameraServer.getInstance().putVideo("Source 2", 160, 120);//CvSink cvSink3 = CameraServer.getInstance().getVideo();//CvSource outputStream3 = CameraServer.getInstance().putVideo("Source 3", 160, 120);//CvSink cvSink4 = CameraServer.getInstance().getVideo();//CvSource outputStream4 = CameraServer.getInstance().putVideo("Source 4", 160, 120);
+      //Mat source1 = new Mat();//Mat source2 = new Mat();//Mat output1 = new Mat();//Mat output2 = new Mat();//Mat source3 = new Mat();//Mat source4 = new Mat();//Mat output3 = new Mat();//Mat output4 = new Mat();
+      /*Mat image1 = new Mat(cvSink1.grabFrame(source1));int height1 = image1.rows();int length1 = image1.cols();MatOfPoint2f src = new MatOfPoint2f(new Point((1/6)*height1, 0),new Point((5/6)*height1,0),new Point(height1-1,length1-1),new Point(0,length1-1));
+      MatOfPoint2f dst = new MatOfPoint2f(new Point(0, 0),new Point(height1-1,0),new Point(0,length1-1),new Point(height1-1,length1-1));Mat hatchWatchMatrix = Imgproc.getPerspectiveTransform(src, dst);*/
       
       while(!Thread.interrupted()) {
-          cvSink1.grabFrame(source1);
-          Imgproc.warpPerspective(source1, output1, hatchWatchMatrix, new Size(height1, length1));
-          //Imgproc.cvtColor(source1, output1, Imgproc.COLOR_BGR2HSV);
-          outputStream.putFrame(output1);
-          cvSink2.grabFrame(source2);
-          //Imgproc.cvtColor(source2, output2, Imgproc.COLOR_BGR2HSV);
-          outputStream2.putFrame(output2);
-          cvSink3.grabFrame(source3);
-          Imgproc.cvtColor(source3, output3, Imgproc.COLOR_BGR2HSV);
-          outputStream3.putFrame(output3);
-          cvSink4.grabFrame(source4);
-          Imgproc.cvtColor(source4, output4, Imgproc.COLOR_BGR2HSV);
-          outputStream4.putFrame(output4);
-  }*/
+          if (seeHatchCam == true){
```

Comment:
Booleans used in `if` statements are typically not compared; they're used as boolean expressions all alone, so this line should just be
```
if (seeHatchCam)
```